spec/spec_helper.rb

```diff
@@ -12,6 +12,10 @@
 Capybara.register_driver :selenium do |app|
   options = Selenium::WebDriver::Chrome::Options.new(
     args: %w(headless disable-gpu no-sandbox --window-size=1440,900 --disable-notifications)
+
+    ### use these args for debugging in chrome.
```

Comment:
I think we could just add the `--remote-debugging` permanently to the args above, and the `Open Chrome` I think we should just pop in the README?